common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/ButtonConstants.kt

```diff
@@ -26,6 +25,8 @@ class ButtonConstants(theme: CurrentTheme) {
 
     val spaceBetween = SpacingScale.xs
     val paddingHorizontal = SpacingScale.md
+    val paddingVertical = 12.dp
+    val paddingVerticalSmall = 6.dp
```

Comment:
cc @mikasdo I wasn't sure if it was appropriate to add a `.padding(.vertical, ..)` to `Button.swift`.

Reply:
I think it's either height, or padding 🤔 but when we have scaling fonts, we can use height as min height and add padding to scaled text when it gets bigger

Reply:
Yeah, the Compose implementation interprets the height as "min height".